rust/src/ldap/types.rs

```diff
@@ -376,7 +376,7 @@ impl<'a> From<ldap_parser::ldap::LdapMessage<'a>> for LdapMessage {
             ldap_parser::ldap::ProtocolOp::IntermediateResponse(msg) => {
                 Self::from_intermediate_response(msg)
             }
-            _ => ProtocolOp::Unknown,
+            ldap_parser::ldap::ProtocolOp::AbandonRequest(_) => ProtocolOp::Unknown,
```

Comment:
Why do we not do anything with this kind of request ?

Reply:
As I reported in a previous PR, I omitted this request because it is not handled correctly by the parser, and it gives an error when it shouldn’t, see here: https://github.com/rusticata/ldap-parser/issues/6

Reply:
Thanks and sorry for the repetition

From what I understand, the rusticate parser errors when it should not.

But when it does not error and returns some `AbandonRequest`, Suricata should still handle it, right ?

Reply:
No pb!
The point is that it always returns an error, and we will never have the abandon request parsed correctly until the parser is fixed.

Reply:
Are you sure it _always_ returns an error ?

Reply:
Yes.

Reply:
Does not look the case to me from the error
```
parsing failed: Error(Ber(UnexpectedClass { expected: Some(ContextSpecific), actual: Application }))
```

Looks like it could return a AbandonRequest with ContextSpecific tags instead of an Application